x-pack/platform/plugins/shared/alerting/public/pages/maintenance_windows/components/table_actions_popover.test.tsx

```diff
@@ -103,4 +133,27 @@ describe('TableActionsPopover', () => {
     fireEvent.click(result.getByTestId('table-actions-icon-button'));
     expect(result.getByTestId('table-actions-unarchive')).toBeInTheDocument();
   });
+
+  test('it shows the success toast when maintenance window id is copied', async () => {
+    const result = appMockRenderer.render(
+      <TableActionsPopover
+        id={'123'}
+        isLoading={false}
+        status={MaintenanceWindowStatus.Archived}
+        onEdit={() => {}}
+        onCancel={() => {}}
+        onArchive={() => {}}
+        onCancelAndArchive={() => {}}
+      />
+    );
+
+    fireEvent.click(result.getByTestId('table-actions-icon-button'));
```

Comment:
According to the docs we should use userEvent to test interaction with our components https://testing-library.com/docs/user-event/intro/#difference-to-fireevent